src/ParseError.js

```diff
@@ -433,6 +433,15 @@ ParseError.EMAIL_MISSING = 204;
  */
 ParseError.EMAIL_NOT_FOUND = 205;
 
+/**
+ * Error code indicating that the email has already been verified and can not be verified again.
+ *
+ * @property {number} EMAIL_ALREADY_VERIFIED
+ * @static
+ */
+ParseError.EMAIL_ALREADY_VERIFIED = 212;
+
```

Comment:
Could you remove the 2nd empty line?

Reply:
sorry for being late. I removed 2nd empty line and push agin.
please check